src/orderbook/Order.py

```diff
@@ -0,0 +1,31 @@
+class Order:
```

Comment:
I disapprove this class for now, dictionnary is fine and will be faster than parsing an object